invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/LoanMetadata/LoanMetadata.js

```diff
@@ -19,6 +27,57 @@ export default class LoanMetadata extends Component {
     this.setState({ isAvailableItemsVisible: newState });
   };
 
+  _renderItem() {
+    const data = this.props.loanDetails;
+    return (
+      !_isEmpty(data.metadata.item) && (
+        <ItemMetadata
+          item={data.metadata.item}
+          loanState={data.metadata.state}
+          changeItemClickHandler={this.showAvailableItems}
+        />
+      )
+    );
+  }
+
+  _renderDeletedItem() {
+    const data = this.props.loanDetails;
+    return (
+      _isEmpty(data.metadata.item) &&
+      data.metadata.item_pid && (
```

Comment:
I like it, thanks! I am just wondering if it is enough the assumption: empty `{}` = deleted item. But for now I think let's keep it like this!